composer.json

```diff
@@ -13,11 +13,18 @@
         }
     ],
 
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/BiffBangPow/Transliterator"
+        }
+    ],
+
     "require": {
         "php": ">=5.3.3",
         "ext-mbstring": "*",
         "behat/gherkin": "^4.4.4",
-        "behat/transliterator": "~1.0",
```

Comment:
@everzet That's done now, and I squashed it all into a single commit. Just waiting on the build.